### ISSUE TITLE
CE-294 Make the love and favorite icons colorblind friendly

### ISSUE THREAD

**Issue by DNin01** [LLK/scratch-www#7137]
### The problem
Currently, colorblind people can have a hard time telling if they have loved/favorited a project because there isn't a major or consistent brightness difference between the inactive and active love/favorite buttons. See this comparison of the buttons when you haven't clicked them and when you have:
![A comparison between the love and favorite icons when you haven't clicked them and when you have clicked them](https://user-images.githubusercontent.com/106490990/197437692-397ba8de-d069-48ad-9fff-ea9ea71aee82.png)
I grayscaled this image and the loved icon was ever so slightly darker than the unloved icon and the favorited icon was a bit brighter than the unfavorited icon.
![The same image but in black and white](https://user-images.githubusercontent.com/106490990/197438258-8c5157b5-9f46-4f8c-9d7c-f90e6446454e.png)

### The solution
Make the inactive love/favorite icons an outline and fill them in when active.
![The top two icons have only a border. The next two icons are filled and colored in.](https://user-images.githubusercontent.com/106490990/197439070-6a9ad771-c02e-4825-8e12-a6b5bf981d6c.png)

**Comment by edelfino11** [comment]
Hi @DNin01 @benjiwheeler I'm a computer science student, and we are currently working towards contributing to an open source GitHub for one of our classes. I would love to take this issue and work towards solving it! Is it ok if I take this issue to work on it? I'm not sure if anyone else is already assigned to this issue, so asking just in case.

**Comment by DNin01** [comment]
> Is it ok if I take this issue to work on it? I'm not sure if anyone else is already assigned to this issue, so asking just in case.

@edelfino11 Don't mind if you do 🙂

**Comment by benjiwheeler** [comment]
@edelfino11 Thanks so much for offering to do this change! We're checking internally with some folks working on accessibility design, and we'll get back to you!

**Comment by edelfino11** [comment]
No problem! @benjiwheeler I figure it might be fixed by simply not filling the svg graphic with the gray color. Also, do you happen to know how to make the main page load properly when I run it locally? When I npm start the repo locally, it looks like this:

<img width="992" alt="Screen Shot 2022-11-27 at 2 24 51 PM" src="https://user-images.githubusercontent.com/55937173/204155683-8083eadd-47b8-473c-b260-a9b2c1efeac7.png">

So I can't check if the change I made fixes the problem. If there's a slack, I can also ask there instead! Sorry for the inconvenience!

**Comment by DNin01** [comment]
Fixed by #7264.
